api/routes/employees.js

```diff
@@ -39,27 +39,49 @@ router.post('/', async (req, res, next) => {
 router.get('/statuses', async (req, res, next) => {
     try {
         const employeesInPostgres = await Employee.findAll({
-            group: ['employee.code', 'employee.fullName', 'employee.active', 'transaction.employeeId', 'transaction.id'],
+            where: {
+                active: true
+            },
+            group: ['employee.code', 'employee.fullName'],
             attributes: [
                 'code',
-                'fullName',
-                'active',
-                [sequelize.fn('max', sequelize.col('transaction.originalDate')), 'lastStatusChange']
-            ],
-            include: [{
-                model: Transaction,
-                attributes: ['originalDate', 'employeeId']
-            }]
+                'fullName'
+            ]
         });
-        const employees = employeesInPostgres.map((e) => ({
-            id: e.code,
-            fullName: e.fullName,
-            lastStatusChange: e.lastStatusChange,
-            status: e.active == 1 ? 'in' : 'out'
-        }));
-        res.json(employees);       
+        const employeesInPostgresPromises = employeesInPostgres.map((e) => {
+            return Transaction.findOne({
+                where: {
+                    employeeId: e.code
+                },
+                attributes: [
+                    'deviceId',
+                    [sequelize.fn('max', sequelize.col('originalDate')), 'lastStatusChange']
+                ],
+                group: [
+                    'deviceId'
+                ],
+                order: [
+                    [sequelize.fn('max', sequelize.col('originalDate')), 'desc']
+                ]
+            }).then((t) => {
```

Comment:
always use variable names that tell something to the reader. Also in case of single function argument the parenthesis should be skipped.